controllers/admin/AdminImportController.php

```diff
@@ -3638,16 +3638,16 @@ public function manufacturerImport($offset = false, $limit = false, $validateOnl
 
     protected function manufacturerImportOne($info, $shop_is_feature_active, $regenerate, $force_ids, $validateOnly = false)
     {
+        if (!$force_ids) {
+            unset($info['id']);
+        }
```

Comment:
This block is duplicated on each method, it could be (in a separate PR of course :sweat_smile: ) in a dedicated method ^^